action.yml

```diff
@@ -1,18 +1,18 @@
-name: 'MongoDB in GitHub Actions'
-description: 'Start a MongoDB server (on default port 27017)'
+name: "MongoDB in GitHub Actions"
+description: "Start a MongoDB server (on default port 27017)"
 
 branding:
-  icon: 'database'
-  color: 'green'
+  icon: "database"
+  color: "green"
 
 inputs:
   mongodb-version:
-    description: 'MongoDB version to use'
+    description: "MongoDB version to use"
     required: false
-    default: 'latest'
+    default: "latest"
 
```

Comment:
Is the REPL_SET input missing here?

Reply:
We use `env`. I'll update the Readme with an example.